modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEDateField.testcase

```diff
@@ -289,7 +289,9 @@ definition {
 			valueMonth = "August",
 			valueYear = 2021);
 
-		Click(locator1 = "Button#PUBLISH");
+		Click(
+			key_text = "Publish",
+			locator1 = "Button#ANY");
```

Comment:
The publish button has changed.
The tests have been tried and the result is in the ticket.